main-settings/src/main/scala/sbt/unchecked.scala

```diff
@@ -0,0 +1,13 @@
+package sbt
+
+import scala.annotation.Annotation
+
+/** An annotation to designate that the annotated entity
+ *  should not be considered for additional sbt compiler checks.
+ *  These checks ensure that the DSL is predictable and prevents
+ *  users from doing dangerous things at the cost of a stricter
+ *  code structure.
+ *
+ *  @since 1.0.0
+ */
+class unchecked extends Annotation
```

Comment:
I don't know if it's a good idea to reintroduce a famous annotation like this. Could we just use the real unchecked for this purpose?